ui/src/utils/markdown.ts

```diff
@@ -39,6 +39,7 @@ export async function render(markdown: string, options: {onlyLink?: boolean, per
 
     md.use(mark)
         .use(meta)
+        .use(mila, {attrs: {target: "_blank", rel: "noopener noreferrer"}})
```

Comment:
Can we restrict these attributes to external links?

```suggestion
        .use(mila, {matcher: (href) => href.match(/^https?:\/\//), attrs: {target: "_blank", rel: "noopener noreferrer"}})
```